modules/widgets.php

```diff
@@ -52,6 +52,20 @@ function jetpack_widgets_add_suffix( $widget_name ) {
 }
 add_filter( 'jetpack_widget_name', 'jetpack_widgets_add_suffix' );
 
-
+/**
+ * Register common assets.
+ *
+ * @since 4.0
+ */
+function jetpack_widgets_init() {
+	// Register minimal grid layout for widgets like My Community
+	wp_register_style(
+		'widget-grid-and-list',
```

Comment:
Whatever we end up doing, we'll need to add the stylesheet to Gruntfile.js as well.